lib/pages/forward/forward_view_style.dart

```diff
@@ -3,9 +3,9 @@ import 'package:flutter/cupertino.dart';
 class ForwardViewStyle {
   static double preferredAppBarSize(BuildContext context) => 64;
 
-  static double get paddingBody => 16.0;
+  static double get paddingBody => 8.0;
 
-  static double get bottomBarHeight => 120.0;
+  static double get bottomBarHeight => 60.0;
 
-  static double get iconSendSize => 40.0;
+  static double get iconSendSize => 56.0;
```

Comment:
```suggestion
  static const int iconSendSize = 56.0;
```
SHould use const to increase build performance